remark-lint-nodejs-yaml-comments.js

```diff
@@ -110,14 +119,13 @@ function validateSecurityChange(file, node, change, index) {
     );
   }
 }
+
 function validateChanges(file, node, changes) {
   if (!Array.isArray(changes))
     return file.message("`changes` must be a YAML list", node);
 
   const changesVersions = [];
-  for (let index = 0; index < changes.length; index++) {
-    const change = changes[index];
-
+  for (const [index, change] of changes.entries()) {
```

Comment:
What's the lint rule behind this change?

Reply:
No rule that's currently used in the repo. I use xo and it flagged this for me.